src/app/components/demo-components.service.ts

```diff
@@ -1161,7 +1161,7 @@ export class SkyDemoComponentsService {
       {
         name: 'Tree view',
```

Comment:
Is “tree view” the correct name for the module and the correct terminology to talk about it? I left it as-is, but we don’t use “view” in the component names or in the naming patterns in the GitHub repo, and it doesn’t seem like the library that we are building off of uses “view” either. Is this really the correct way to refer to this module? And if so, why isn’t this wording convention built into the names of the components and such? (The Development tab had multiple instance of just “tree” instead of “tree view” and I updated them for consistency, but I’m not sure this change that was really the right call.)

Reply:
Yes, we have a consistency problem here. Our pattern for 3rd party library components is to use the library's name. The only thing I don't care for in this particular scenario is that the third party library has the words "Angular" and "component" in its name. Bleh. With that said, I think we should probably still follow the pattern to avoid confusion. After all, this is our SKY-specific implementation of `angular-tree-component`. Here's some examples of how I think we should refer to it:

The npm package:
`@skyux/angular-tree-component`

The repo:
`skyux-angular-tree-component`

The file directory:
`/angular-tree-component/...`

How we *should* refer to the component:
`Angular tree component`

What do you think?

Reply:
It makes sense to me to follow the pattern of using the third-party library's name, and I'm good with all these naming changes. But I think we should confirm this with @Blackbaud-ToddRoberts first to make sure he's OK with it.

Reply:
IMO the info architecture/nav should be implementation-agnostic both for ease of discovery (users would be looking for "Tree view" not "Angular tree component") and future-proofing if we ever changed implementation or added additional implementations (e.g. native mobile components)

Reply:
So then for the examples Alex provides above, should we go with:

The npm package:
`@skyux/tree-view`

The repo:
`skyux-tree-view`

The file directory:
`/tree-view/..`.

How we should refer to the component:
`tree view component`

Reply:
Repo and npm package I am not worried about, it's probably valuable to be implementation-specific there. It's more in the docs navigation and pattern-related (i.e. non-implementation-related) content that we should be agnostic.

Reply:
Todd makes a good point about how we should lead with the UX pattern, and not the implementation. That makes me think `tree view` would be better for the navigation of the docs site and the title of the page. However, the repo, npm package, and component should still be called `angular-tree-component` because those ARE implementation-specific. We wouldn't add a "react" component to the same package. It would just be a new package - for example, `@skyux/react-super-treeview` (an actual react open-source library).

Reply:
FWIW - we'll have this same problem with the ag-Grid pr: https://github.com/blackbaud/skyux2-docs/pull/711

Reply:
OK. Based on this decision, I think most of the docs are OK as is, but I'll take another pass to make sure.

Reply:
I went through, and the docs seem OK. ... For the naming patterns, should "angular" be part of the file names and folder paths for the demos?

Reply:
@Blackbaud-AlexKingman: Any thoughts on the file names and folders for the demos? Should they include "angular"?

Reply:
Hmm... the folder names are exposed to the end user, so I feel like they should match our descriptors on the docs page. If we refer to this page as `Tree view`, then the folder should probably be `./tree-view` as well.

Following this pattern, you can make an argument that our demo is a specific implementation. So perhaps the demo files should match the 3rd-party library name. For example:
```
./angular-tree-component/angular-tree-component-demo.component.html
./angular-tree-component/angular-tree-component-demo.component.ts
./angular-tree-component/angular-tree-component-demo.module
```

What do you think?

Reply:
demo files matching 3rd party library name seems correct to me.

Reply:
I tried to update the demo filenames but ran into errors. I spent some time troubleshooting but just got different errors. Rather than spin my wheels troubleshooting, Alex and I decided that he would just make those changes in his main docs branch. I'm resolving this conversation since we'll address the filenames there.